### PR TITLE
Changed the styling of github button

### PR DESCRIPTION
**Pull request by snehafarkya**
Hey! I have upgraded the styling of the GitHub button. 

Kindly view it and accept it.

### CODE REVIEW

**Comment by dt2patel** [comment]
Related issue is not requested, PR can be accepted for using flex instead of grid, but still need to consider below points.

A few things to think about @snehafarkya:
Code indentation?
What is the default flex direction?

**Comment by snehafarkya** [comment]
Ohh sure. Can you review it now?

File: index.html [hunk 1]
```diff
@@ -420,17 +421,18 @@ <H1>Contact Us</H1>
   }
 
   .tag{
-    display: grid;
-    grid-auto-flow: column;
+    display: flex;
     grid-gap: 8px;
```
Comment (dt2patel) [inline index.html:425]: Need to update property to "gap" since we are not using grid anymore

File: index.html [hunk 2]
```diff
@@ -420,17 +421,18 @@ <H1>Contact Us</H1>
...
     width: max-content;
     align-items: center;
-    border-radius: 16px;
-    padding: 0 8px;
+    justify-content: center;
+    border-radius: 12px;
```
Comment (dt2patel) [inline index.html:429]: remove border radius and padding to maintain original styling.

File: index.html [hunk 3]
```diff
@@ -420,17 +421,18 @@ <H1>Contact Us</H1>
...
     margin: auto;
     margin-top: 8px;
     color: var(--surface);
     text-transform: uppercase;
+    padding: 0.3em 1em ;
+
```
Comment (dt2patel) [inline index.html:435]: remove extra space.

**Review by dt2patel — CHANGES_REQUESTED** [review]
almost ready to merge. please see comments

**Pull request closed without merging**